porter/v1/cluster_control_plane.proto

```diff
@@ -38,7 +38,10 @@ message CreateAzureConnectionRequest {
   bytes service_principal_secret = 5;
 }
 
-message CreateAzureConnectionResponse {}
+message CreateAzureConnectionResponse {
```

Comment:
From the perspective of an API consumer, does this mean that the response will be a connection pool for azure?

Reply:
I dont see this being used in the service either

Reply:
It's really just saving the credentials, so I'll go ahead and rename it to `SaveAzureCredentials`